### DISCUSSION TITLE
Can't opml export

### DISCUSSION THREAD

**Discussion by RandomBloke85** [FreshRSS/FreshRSS#5416] [Q&A] [answered]
Hello

I can't find the opml export in the settings menu. What am I doing wrong? Please excuse the basic question, but it seems to be gone. (version 1.21.1-dev).

Thank you

**Comment by Alkarex** [comment, marked as answer]
1) From "subscription management"
![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/4a4f1f7d-ca32-4aa6-a038-c294beff37af)

2) Import / Export
![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/bf8ae41e-589a-46e8-a729-9ee139aab3c2)

Bonus)
From any view, e.g. viewing a category, an OPML for that view can be generated by using `a=opml` in the URL parameters, for instance `https://freshrss.example.net/i/?a=opml&get=c_11`
This is declared in the HTML headers

```html
<link rel="outline" type="text/x-opml" title="OPML" href="./?a=opml&amp;get=c_11" />
```

**Comment by RandomBloke85** [comment]
Ok, I didn't look there since I thought it should be in the settings panel. My bad. Sorry. Please close, or better delete that topic. Thank you

**Reply by Alkarex** [reply]
No worries, it might be useful for others
